libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Lenses.hs

```diff
@@ -47,6 +53,9 @@ mmL = lens mm (\ds u -> ds { mm = u })
 
 -}
 
+updateWithLens :: Lens' a b -> (b -> b) -> a -> a
+updateWithLens l f a = a & l .~ f (a ^. l)
```

Comment:
There is already an operator for this: [`%~`](https://hackage.haskell.org/package/microlens-0.4.13.1/docs/Lens-Micro.html#v:-37--126-)
```suggestion
```